internal/pkg/gateway/registry/registry.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2021 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package registry
+
+import (
+	"crypto/x509"
+	"fmt"
+	"github.com/hyperledger/fabric-protos-go/orderer"
+	"github.com/hyperledger/fabric-protos-go/peer"
+	"github.com/hyperledger/fabric/common/flogging"
+	pp "github.com/hyperledger/fabric/core/peer"
+	"github.com/hyperledger/fabric/discovery"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+// Config represents the startup configuration of the gateway
+type Config interface {
```

Comment:
What is this providing? Why would it be an interface?

Reply:
I've removed this for now